common/config/config.go

```diff
@@ -38,15 +38,18 @@ const (
 	SKIP_VERSION_VERIFY       = "LAYER0_SKIP_VERSION_VERIFY"
 	TEST_AWS_TAG_DYNAMO_TABLE = "LAYER0_TEST_AWS_TAG_DYNAMO_TABLE"
 	TEST_AWS_JOB_DYNAMO_TABLE = "LAYER0_TEST_AWS_JOB_DYNAMO_TABLE"
+	AWS_TIME_BETWEEN_REQUESTS = "LAYER0_AWS_TIME_BETWEEN_REQUESTS"
 )
 
 // defaults
 // bGF5ZXIwOm5vaGF4cGx6 = layer0:nohaxplz, base64 encoded (basic http auth)
 const (
-	DEFAULT_AUTH_TOKEN   = "bGF5ZXIwOm5vaGF4cGx6"
-	DEFAULT_API_ENDPOINT = "http://localhost:9090/"
-	DEFAULT_API_PORT     = "9090"
-	DEFAULT_AWS_REGION   = "us-west-2"
+	DEFAULT_AUTH_TOKEN            = "bGF5ZXIwOm5vaGF4cGx6"
+	DEFAULT_API_ENDPOINT          = "http://localhost:9090/"
+	DEFAULT_API_PORT              = "9090"
+	DEFAULT_AWS_REGION            = "us-west-2"
```

Comment:
how does this work when the user specifies a different region as part of `l0-setup`?

Reply:
(i.e. does the region always get set properly as an env var)

Reply:
It is either passed in as a CLI flag at setup time or uses the default:

https://github.com/quintilesims/layer0/blob/466-api-retry-delay-logic-jparsons/setup/command/aws.go#L59

Reply:
Additionally, if `LAYER0_AWS_REGION` is set as an environment variable, then the `AWSRegion()` function will pull that

https://github.com/quintilesims/layer0/blob/466-api-retry-delay-logic-jparsons/common/config/config.go#L156

Reply:
right, but this is problematic since `l0-setup endpoint` (as well as `-d`) does not include the aws region in its' list of exports. We need to add that otherwise the default region of `us-west-2` will always be used.